sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -327,7 +332,7 @@ export const enum KnownKeyCurveNames {
 }
 
 // @public
-export const enum KnownKeyOperations {
+export enum KnownKeyOperations {
```

Comment:
We should avoid `const enum` whenever possible, not breaking as the net-effect is the same for TS and additive for JS. I plan to incrementally replace `const enum` with `enum` where I can. I did not have lot of success with `preserveConstEnums` though